### DISCUSSION TITLE
[Feature]: Allow scroll of the canvas if the device height exceeds the available height

### DISCUSSION THREAD

**Discussion by grant007** [GrapesJS/grapesjs#5296] [Roadmap Suggestions]
Given the following code example...

```
editor.DeviceManager.add('test', '300px', {
                      height: '1000px',
                      widthMedia: '300px',
});
editor.setDevice("test");

```
Results in no ability to scroll to top of canvas if editor display area is less than specified height of the device.

See Fiddle for demonstration...

https://jsfiddle.net/gstone007/0vq3o2jf/1/

**Comment by ghost** [comment]
Just add style `overflow: auto` to .gjs-cv-canvas__frames

**Comment by artf** [comment]
After exploring a bit more about this issue I'd suggest just not to exceed the canvas height with the device one. Unfortunately, currently, making any part of the canvas scrollable (solution suggested by @TheDiamondKing ) will lead to unexpected behaviors. 
I'm working on the frame and canvas refactoring, so probably this will be solved differently

**Comment by contentfree** [comment]
@artf Is your comment still true? Does adding `overflow: auto` (especially in a "designer mode" / "absolute mode" canvas) still cause problems?
